### ISSUE TITLE
Add TC74 i2c temperature sensor library

### ISSUE THREAD

**Issue by iayanpahwa** [adafruit/circuitpython#2233]
TC74 from microchip is a cheap and good i2c based temperature sensor, will be good to have a supported library in CircuitPython

**Comment by ladyada** [comment]
this will probably work https://github.com/adafruit/Adafruit_CircuitPython_PCT2075

**Comment by iayanpahwa** [comment]
Thanks @ladyada , I'll give it a try .

**Comment by ladyada** [comment]
@siddacious @kattni plz update the library readme to clarify it will also work on the TC74 :)

**Comment by iayanpahwa** [comment]
@ladyada do you think we can update the library to be generic and used with both. Not sure but TC74 supports low power modes, can add function to utilise that, I can work on it. :)

**Comment by ladyada** [comment]
thats right, some things may vary, please check the TC74 and PCT2075 datasheet, i believe PCT has everything TC74 does but please add any functions!

**Comment by iayanpahwa** [comment]
sure

**Comment by siddacious** [comment]
@ladyada README updated!
